app/main/hotkeys.js

```diff
@@ -29,6 +29,8 @@ const HOTKEYS = new Map([
   ['CmdOrCtrl+6', bw => () => bw.webContents.send('select-tab-index', 5)],
   ['CmdOrCtrl+7', bw => () => bw.webContents.send('select-tab-index', 6)],
   ['CmdOrCtrl+8', bw => () => bw.webContents.send('select-tab-index', 7)],
+  ['CmdOrCtrl+Alt+Left', bw => () => bw.webContents.send('select-tab-left')],
```

Comment:
My first impression was that the actions should be named 'previous' / 'next' instead of 'left' / 'right' because we may visually change tab layout somehow